docs/source/app_developers_guide/aws.rst

```diff
@@ -28,13 +28,14 @@ to use an SDK to create a transaction family that implements your application's
 business logic.
 
 
-About the Application Development Environment
-=============================================
-
-The AWS application development environment is a single validator node that is
-running a validator, a REST API, and three transaction processors. This
-environment uses Developer mode (dev mode) consensus and serial transaction
-processing.
+About the AWS Test Node Environment
+===================================
+
+This AWS environment is a single Sawtooth node that is running a
+validator, a REST API, the Devmode consensus engine, and three transaction
+processors. The environment uses simple
+:ref:`Devmode consensus <dynamic-consensus-label>` and
+:doc:`parallel transaction processing <../architecture/scheduling>`.
```

Comment:
This commit says it uses parallel; before it said is uses serial. If this change is intentional, please add something to the commit indicating this correction.

Reply:
Good catch - thanks!  The default changed from serial to parallel a while back, but I don't know if AWS picked up that change or not.  Some investigation is needed.

Reply:
The documentation for AWS images was intentionally not changed to parallel along with the rest of the system because the AWS images had not been updated to a version which defaulted to parallel. I believe that is still the case. @rbuysse or @rberg2 should be able to confirm.

Reply:
Intel has published a Sawtooth AMI version 1.1.4 [here](https://aws.amazon.com/marketplace/pp/B075TKQCC2)

Reply:
Interesting. 

For this PR, I'll change "parallel" back to the original "serial".  If the AWS procedure needs to change as a result of Sawtooth AMI 1.1.4, that would need to be done in a different PR.

Reply:
Fixed. (Changed "parallel" back to "serial" in the AWS procedure.)